README.md

```diff
@@ -9,70 +9,8 @@ for the purpose of allowing programs to be extracted and compiled from JAX
 for deployment with IREE, without a Python or JAX dependency.
 
 In order to understand how this works, it is important to have a working
-knowledge of what JAX and IREE programs are.
-
-## What is a JAX program?
-
-The challenge with talking about extracting a JAX program is that in all
-generality, a JAX program is bounded only by what can run in the host Python
-interpreter. What we are looking for is a simpler definition that allows a
-useful set of standalone programs to be constructed and that meshes well with
-the programming model employed by typical JAX developers.
-
-The components that are the most interesting towards this end are:
-
-* `jax.jit` functions: stateless functions mapping inputs to outputs, natively
-  represented as JAXPR IR and universally convertible to MHLO IR.
```

Comment:
I'd like to unpack this as a newbie:
1. Is JAXPR an intermediate representation in MLIR or its own IR unbound to MLIR?
2. Is the path of lowering always going to be JAXPR IR -> MHLO IR or are/will there be other IRs?
3. What does "universally convertible" mean?